common/script_values/00_men_at_arms_values.txt

```diff
@@ -365,3 +315,177 @@ culture_ai_weight_heavy_cavalry = {
+		limit = {
+			OR = {
+				culture_group = culture_group:human_group
+				culture_group = culture_group:gnome_group
+			}
+		}
+		add = -60
+	}
+	
+	if = { # Cultures with a special MaA of that type should avoid the base version of the unit
+		limit = {
+			culture = {
+				OR = {
+
+				}
+			}
+		}
+		add = -100
+	}
+}
+culture_ai_weight_light_aerial = {
+	value = 0
+	
+	if = { # Cultures that favours light aerial
+		limit = {
+			OR = {
+				culture_group = culture_group:human_group
+				culture_group = culture_group:elf_group
+				culture_group = culture_group:dwarf_group
+				culture_group = culture_group:harpy_group
+				culture_group = culture_group:dragon_group
+			}
+		}
+		add = 40
+	}
+	if = { # Cultures that dislike light aerial
+		limit = {
+			OR = {
+				culture_group = culture_group:hozen_group
+				culture_group = culture_group:kobold_group
+				culture_group = culture_group:murloc_group
+				culture_group = culture_group:trogg_group 
+				culture_group = culture_group:gnoll_group
+				culture_group = culture_group:magnataur_group
+				culture_group = culture_group:tuskarr_group
+				culture_group = culture_group:gnoll_group
+				culture_group = culture_group:centaur_group
+				culture_group = culture_group:quilboar_group
+				culture_group = culture_group:furbolg_group
+				culture_group = culture_group:pygmy_group
+				culture = culture:blood_troll
+				culture = culture:yaungol
+				culture = culture:drust
+				culture = culture:giblin
+				culture = culture:saurok
+			}
+		}
+		add = -60
+	}
+	
+	if = { # Cultures with a special MaA of that type should avoid the base version of the unit
+		limit = {
+			culture = {
+				OR = {
+
+				}
+			}
+		}
+		add = -100
+	}
+}
+culture_ai_weight_massive_aerial = {
+	value = 0
+	
+	if = { # Cultures that favours massive aerial
+		limit = {
+			OR = {
+				culture_group = culture_group:gnome_group
+				culture_group = culture_group:dragon_group
+				culture = culture:vrykul
+				culture = culture:kvaldir
+				culture = culture:hyldnir
+				culture = culture:frost_vrykul
+				culture = culture:goblin
+			}
+		}
+		add = 40
+	}
+	if = { # Cultures that dislike massive aerial
+		limit = {
+			OR = {
+				culture_group = culture_group:hozen_group
+				culture_group = culture_group:kobold_group
+				culture_group = culture_group:murloc_group
+				culture_group = culture_group:trogg_group 
+				culture_group = culture_group:gnoll_group
+				culture_group = culture_group:magnataur_group
+				culture_group = culture_group:tuskarr_group
+				culture_group = culture_group:gnoll_group
+				culture_group = culture_group:centaur_group
+				culture_group = culture_group:quilboar_group
+				culture_group = culture_group:furbolg_group
+				culture_group = culture_group:pygmy_group
+				culture_group = culture_group:harpy_group
+				culture = culture:blood_troll
+				culture = culture:yaungol
+				culture = culture:drust
+				culture = culture:giblin
+				culture = culture:saurok
+			
+			}
```

Comment:
You should add a scripted trigger like `dislike_aerial_trigger` because you used the same cultures in `culture_ai_weight_light_aerial`.